File: MultiplesOf3or5/solution.c
int solution(int number) {
    if (number < 0) {
        return -1;
    }

    int total = 0;

    for (int i = 1; i < number; i++) {
        if (i % 3 == 0 && i % 5 == 0) {
            total += i;
        }
    }

    return total;
}

File: MultiplesOf3or5/solution.r
solution <- function(number) {
    if (number < 0) {
        return(0);
    }

    total <- 0;

    for (i in 1:number-1) {
        if (i %% 3 == 0 | i %% 5 == 0) {
            total <- total + i;
        }
    }

    return(total);
}